src/r5py/r5/travel_time_matrix_computer.py

```diff
@@ -92,10 +100,24 @@ def __init__(
             transport_network = TransportNetwork(*transport_network)
         self.transport_network = transport_network
 
+        # Quick validation to ensure an ID column exists
+        if "id" not in origins.columns:
```

Comment:
As discussed in the issue, I would perhaps also add the possibility to specify a column which is used as unique id. It's a bit silly restriction to always expect 'id' column to be found as it can be called anything.

Reply:
I guess a separate issue or PR would be a good place :+1:

Reply:
Yep, sounds good 👍🏻

Reply:
Thinking about it, I would argue against a configurable column name. It’s straight-forward for the user to rename a column, at the same time the code to handle dynamic column names necessarily will be fairly bloated. With good documentation, requiring specific column names is a very pythonic way of handling this.

Reply:
I guess this comment is resolved from this PR. Whether or not to allow configurable column name can be discussed in a separate issue / PR.